udata/tests/dataset/test_dataset_tasks.py

```diff
@@ -22,6 +23,9 @@ def test_purge_datasets():
         Dataset.objects.create(title="keep me"),
     ]
 
+    # This is needed or the test will fail if you have a search service configured in your udata.cfg file.
+    current_app.config["SEARCH_SERVICE_API_URL"] = ""
+
```

Comment:
This is unrelated to this PR, but this test will fail if you have a search service configured, which I needed to fix this PR ;)